presets/4.5/bnf/droneaid.txt

```diff
@@ -115,6 +114,11 @@ vtx 2 2 0 0 4 1900 2100
 
 #$ OPTION_GROUP END
 
+# vtx
+vtx 0 2 0 0 1 900 1100
+vtx 1 2 0 0 2 1400 1600
+vtx 2 2 0 0 4 1900 2100
```

Comment:
Some of the VTXes have 5 power levels, some 4, so that's why we added this section before all the VTX settings to be able to overwrite them with the VTX-specific one.

Reply:
You can find the example of this here:
https://github.com/vbereshchenko/droneaid-firmware-presets/blob/master/presets/4.5/bnf/droneaid.txt#L77